pkg/ibmvpc/analysis_output_test.go

```diff
@@ -570,7 +572,7 @@ func getVPCConfigs(t *testing.T, tt *vpcGeneralTest, firstCfg bool) map[string]*
 	} else {
 		inputConfig = tt.inputConfig2nd
 	}
-	inputConfigFile := filepath.Join(getTestsDir(), inputConfig)
+	inputConfigFile := filepath.Join(getTestsDir("input"), inputConfig)
```

Comment:
better have a separate function instead of repeating `filepath.Join(getTestsDir("input"), ...` with hardcoded `input` dir prefix (and also define it as const)